docs/UserGuide.adoc

```diff
@@ -275,23 +275,31 @@ Clocks out for the employee John Doe.
 
 ==== Add Order
 
+Important Terminology
```

Comment:
This should be in its own section that explains all four terminologies before the list of commands section